content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -89,7 +89,13 @@ Otherwise, download and run the [Windows installer](https://storage.googleapis.c
 
 {{% capture whatsnext %}}
 
-* [Running Kubernetes Locally via Minikube](/docs/setup/learning-environment/minikube/)
+* [Running Kubernetes locally via minikube](/docs/setup/learning-environment/minikube/)
+
+If you would like additional help to get started, please see:
+
```

Comment:
Hi @tstromberg .
Not sure what happened to the order of the content.
I would expect all of the steps to be listed within the `capture steps` directive.
I don't think moving the `help starting minikube` section into the `What's next` section works as well.

Reply:
I thought the order was kind of weird too. I've addressed it and simplified the text some. PTAL.